kong/clustering/rpc/utils.lua

```diff
@@ -23,4 +28,16 @@ function _M.is_timeout(err)
 end
 
 
+function _M.compress_payload(payload)
+  local json = assert(cjson_encode(payload))
+  return assert(snappy_compress(json))
```

Comment:
Is it a tail call? should we rewrite it like:
```
local data = assert(snappy_compress(json))
return data
```

Reply:
Addressed